crates/compiler/fmt/src/expr.rs

```diff
@@ -1018,7 +1051,7 @@ pub fn expr_lift_spaces<'a, 'b: 'a>(
 ) -> Spaces<'a, Expr<'a>> {
     match expr {
         Expr::Apply(func, args, called_via) => {
```

Comment:
This may need some more adjustment. I think you probably want to just write a separate branch for when we have `CalledVia::ParensAndCommas`, and that will look fairly different. Specifically, I think you'll just want to do expr_lift_spaces_before(func), and then not touch the args at all since all their spaces will be inside the parens and thus don't need to be "lifted".